spec/appcenter_upload_spec.rb

```diff
@@ -1498,6 +1498,54 @@ def stub_add_to_destination(status, app_name = "app", owner_name = "owner", dest
       expect(values[:dsym_path]).to eq('./spec/fixtures/symbols/Themoji.dSYM.zip')
     end
 
+    it "allows to upload build only even if dsym provided when upload_build_only is true"  do
```

Comment:
spec/appcenter_upload_spec.rb:1501:90: C: Layout/ExtraSpacing: Unnecessary spacing detected.
    it "allows to upload build only even if dsym provided when upload_build_only is true"  do

Reply:
fixed